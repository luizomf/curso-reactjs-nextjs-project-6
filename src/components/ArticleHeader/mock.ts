import { ArticleHeaderProps } from '.';
import { data } from '../../api/dados.json';

const { title, excerpt, cover, author, categories, createdAt } = data.posts[0];

export default {
  title,
  excerpt,
  cover,
  author,
  categories,
  createdAt,
} as ArticleHeaderProps;
