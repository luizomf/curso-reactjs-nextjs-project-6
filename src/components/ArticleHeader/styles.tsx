import styled, { css } from 'styled-components';
import { Title as HeadingStyles } from '../Heading/styles';

export const Wrapper = styled.header`
  ${({ theme }) => css`
    padding-bottom: ${theme.spacings.xlarge};
    margin-bottom: ${theme.spacings.xlarge};
    border-bottom: 0.1rem solid ${theme.colors.mediumGray};

    ${HeadingStyles} {
      margin: 0;
      margin-bottom: ${theme.spacings.medium};
    }
  `}
`;

export const Excerpt = styled.p`
  ${({ theme }) => css`
    margin: ${theme.spacings.medium} 0;
    font-size: ${theme.font.sizes.medium};
  `}
`;

export const Cover = styled.img`
  ${({ theme }) => css`
    max-width: 100%;
    display: block;
    margin-bottom: ${theme.spacings.medium};
  `}
`;
