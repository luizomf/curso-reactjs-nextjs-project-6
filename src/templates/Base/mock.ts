import { BaseTemplateProps } from '.';
import { data } from '../../api/dados.json';

export default {
  settings: data.setting,
  children: 'Ol√° mundo',
} as BaseTemplateProps;
