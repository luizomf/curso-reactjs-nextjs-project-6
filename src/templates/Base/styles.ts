import styled, { css } from 'styled-components';

export const Wrapper = styled.div`
  svg.search-ok-icon,
  svg.search-cancel-icon {
    width: 2.4rem;
    height: 2.4rem;
    margin-left: 1rem;
  }

  svg.search-cancel-icon {
    ${({ theme }) => css`
      color: ${theme.colors.secondary};
    `}
  }
`;

export const HeaderContainer = styled.div`
  ${({ theme }) => css`
    margin-top: ${theme.spacings.xhuge};
    padding: 0 ${theme.spacings.large};
  `}
`;

export const SearchContainer = styled.div`
  ${({ theme }) => css`
    margin: 0 auto;
    margin-bottom: ${theme.spacings.xlarge};
    padding: 0 ${theme.spacings.large};
    max-width: 120rem;
    width: 100%;
    display: flex;
    align-items: center;
    justify-content: center;
  `}
`;

export const SearchInput = styled.input`
  ${({ theme }) => css`
    padding: 0.5rem ${theme.spacings.small};

    &:disabled {
      outline: 0.1rem solid ${theme.colors.secondary};
    }
  `}
`;

export const ContentContainer = styled.div`
  ${({ theme }) => css`
    max-width: 120rem;
    width: 100%;
    margin: 0 auto;
    margin-bottom: ${theme.spacings.large};
  `}
`;

export const FooterContainer = styled.div`
  ${({ theme }) => css`
    max-width: 120rem;
    width: 100%;
    margin: 0 auto;
    padding: ${theme.spacings.large};
  `}
`;
