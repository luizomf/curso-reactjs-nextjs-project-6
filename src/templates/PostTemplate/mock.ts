import { PostTemplateProps } from '.';
import { data } from '../../api/dados.json';

export default {
  settings: data.setting,
  post: data.posts[0],
} as PostTemplateProps;
