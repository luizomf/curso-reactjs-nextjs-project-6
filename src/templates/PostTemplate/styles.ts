import styled, { css } from 'styled-components';

export const TagsContainer = styled.div`
  ${({ theme }) => css`
    padding: 0 ${theme.spacings.large};
  `}
`;
