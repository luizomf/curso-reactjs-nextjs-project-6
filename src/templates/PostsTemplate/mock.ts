import { PostsTemplateProps } from '.';
import { data } from '../../api/dados.json';

export default {
  settings: data.setting,
  posts: data.posts,
} as PostsTemplateProps;
