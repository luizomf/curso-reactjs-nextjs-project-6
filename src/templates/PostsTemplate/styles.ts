import styled, { css } from 'styled-components';

export const ButtonContainer = styled.div`
  ${({ theme }) => css`
    padding: 0 ${theme.spacings.large};
    margin: ${theme.spacings.large} 0;
    display: flex;
    justify-content: center;
  `}
`;

export const Button = styled.button`
  ${({ theme }) => css`
    background: ${theme.colors.primary};
    color: ${theme.colors.white};
    border: none;
    padding: ${theme.spacings.small} ${theme.spacings.large};
    cursor: pointer;

    &:disabled {
      background: ${theme.colors.darkerGray};
    }
  `}
`;
