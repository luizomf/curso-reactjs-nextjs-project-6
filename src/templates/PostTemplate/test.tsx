import { renderTheme } from '../../styles/render-theme';
import { PostTemplate, PostTemplateProps } from '.';

import mock from './mock';

const props: PostTemplateProps = mock;

describe('<PostTemplate />', () => {
  it('should match snapshot', () => {
    const { container } = renderTheme(<PostTemplate {...props} />);

    expect(container).toMatchSnapshot();
  });
});
