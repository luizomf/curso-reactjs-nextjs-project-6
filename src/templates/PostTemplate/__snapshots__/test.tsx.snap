// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PostTemplate /> should match snapshot 1`] = `
.c22 {
  padding: 0 3.2rem;
}

.c19 {
  font-size: 1.6rem;
  color: #AAAAAA;
  font-style: italic;
}

.c19 .categories span::after {
  content: ', ';
}

.c19 .categories span:last-child::after {
  content: '';
}

.c19 a {
  color: #dc143c;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c19 a:hover {
  -webkit-filter: brightness(50%);
  filter: brightness(50%);
}

.c6 {
  color: #000000;
  line-height: 1.2;
  font-size: 2.4rem;
  text-transform: uppercase;
}

.c12 {
  color: #000000;
  line-height: 1.2;
  font-size: 2.4rem;
  text-transform: none;
}

.c16 {
  color: #000000;
  line-height: 1.2;
  font-size: 6.4rem;
  text-transform: none;
}

.c15 {
  padding-bottom: 4.0rem;
  margin-bottom: 4.0rem;
  border-bottom: 0.1rem solid #DDDDDD;
}

.c15 .c5 {
  margin: 0;
  margin-bottom: 2.4rem;
}

.c17 {
  margin: 2.4rem 0;
  font-size: 2.4rem;
}

.c18 {
  max-width: 100%;
  display: block;
  margin-bottom: 2.4rem;
}

.c21 {
  font-size: calc(1.6rem + 0.2rem);
  line-height: 1.5;
}

.c21 p {
  margin: 2.4rem 0;
}

.c21 a,
.c21 a:visited,
.c21 a:link {
  color: #dc143c;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c21 a:hover {
  -webkit-filter: brightness(50%);
  filter: brightness(50%);
}

.c21 code {
  font-family: monospace;
  color: #dc143c;
  font-size: 1.6rem;
  background: #DDDDDD;
  padding: 0.2rem;
  margin: 0.2rem;
}

.c21 pre {
  background: #000000;
  padding: 2.4rem;
  font-family: monospace;
  color: #FFFFFF;
  margin: 2.4rem 0;
  width: 100%;
  overflow-x: auto;
  font-size: 1.6rem;
}

.c21 pre code {
  color: inherit;
  background: inherit;
}

.c21 img {
  max-width: 100%;
}

.c21 .image {
  background: #DDDDDD;
  line-height: 0;
  margin: 2.4rem 0;
}

.c21 .image figcaption {
  font-size: 1.6rem;
  padding: 1.6rem;
  text-align: center;
  line-height: 1.3;
}

.c21 .image-style-side {
  max-width: 50%;
  float: right;
  margin: 2.4rem 0;
}

.c21 hr {
  border: none;
  border-bottom: 0.1rem solid #DDDDDD;
}

.c21 ul,
.c21 ol {
  margin: 2.4rem 4.0rem;
}

.c21 .table {
  width: 100%;
  overflow: hidden;
  overflow-x: auto;
}

.c21 table {
  width: 100%;
  border-collapse: collapse;
  margin: 2.4rem 0;
}

.c21 table td,
.c21 table th {
  padding: 1.6rem;
  border: 0.1rem solid #DDDDDD;
}

.c21 blockquote {
  border-left: 0.5rem solid #dc143c;
  color: #AAAAAA;
  -webkit-filter: brightness(80%);
  filter: brightness(80%);
  padding-left: 2.4rem;
  font-style: italic;
  margin: 2.4rem;
}

.c14 {
  width: 100%;
  margin: 0 auto;
  padding: 0 3.2rem;
  max-width: 96rem;
}

.c20 {
  width: 100%;
  margin: 0 auto;
  padding: 0 3.2rem;
  max-width: 80rem;
}

.c23 {
  margin: 2.4rem 0;
}

.c23 span {
  margin: 0 0 0 0.5rem;
}

.c23 span::before {
  content: ' ';
}

.c23 span::after {
  content: ', ';
}

.c23 span:last-child::after {
  content: '';
}

.c23 a {
  color: #dc143c;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c23 a:hover {
  -webkit-filter: brightness(50%);
  filter: brightness(50%);
}

.c25 {
  text-align: center;
  border-top: 0.1rem solid #DDDDDD;
}

.c25 a {
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c26 {
  position: fixed;
  background: #000000;
  color: #FFFFFF;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 4rem;
  height: 4rem;
  bottom: 2rem;
  right: 2rem;
  z-index: 6;
}

.c1 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
}

.c7 > img {
  width: 15rem;
  height: 15rem;
  border-radius: 50%;
}

.c10 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding-bottom: 6.4rem;
  width: 100%;
  max-width: 96rem;
  color: #333333;
  font-size: 1.6rem;
  margin: 0 auto;
}

.c10 .c5 {
  margin: 0 0 calc(1.6rem - 1rem);
}

.c11 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column wrap;
  -ms-flex-flow: column wrap;
  flex-flow: column wrap;
  margin-left: 3.2rem;
  max-width: 48rem;
}

.c8 {
  display: block;
  color: #FFFFFF;
  -webkit-text-decoration: none;
  text-decoration: none;
  margin-bottom: 1.6rem;
  font-size: 1.8rem;
  border-right: 0.5rem solid #000000;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c8:hover {
  border-right: 0.5rem solid #dc143c;
  color: #dc143c;
}

.c2 {
  background: #000000;
  padding: 3.2rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  position: fixed;
  z-index: 1;
  width: 100%;
  max-width: 32rem;
  height: 100vh;
  top: 0;
  left: 0;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
  overflow-y: auto;
  left: -30rem;
  overflow-y: hidden;
}

.c3 {
  margin: auto;
  width: 100%;
}

.c4 .c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin: 0;
  margin-bottom: 4.8rem;
}

.c4 .c5 img {
  border: 0.5rem solid #dc143c;
}

.c0 {
  position: fixed;
  top: 2.4rem;
  color: #FFFFFF;
  background: #000000;
  z-index: 2;
  width: 3rem;
  height: 3rem;
  left: 26rem;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
  left: 1rem;
  color: #FFFFFF;
}

.c9 {
  margin-top: 6.4rem;
  padding: 0 3.2rem;
}

.c13 {
  max-width: 120rem;
  width: 100%;
  margin: 0 auto;
  margin-bottom: 6.4rem;
}

.c24 {
  max-width: 120rem;
  width: 100%;
  margin: 0 auto;
  padding: 3.2rem;
}

@media (max-width:768px) {
  .c16 {
    font-size: 4.0rem;
  }
}

@media (max-width:768px) {
  .c21 {
    font-size: 2rem;
  }

  .c21 .image-style-side {
    max-width: 100%;
    float: none;
    margin: 0;
  }
}

@media (max-width:360px) {
  .c10 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-flex-flow: column wrap;
    -ms-flex-flow: column wrap;
    flex-flow: column wrap;
  }

  .c10 > .c5 {
    margin: 0 0 2.4rem 0;
  }
}

@media (max-width:360px) {
  .c11 {
    margin-left: 0;
  }
}

@media (max-width:768px) {
  .c2 {
    left: -32rem;
  }
}

@media (max-width:768px) {
  .c0 {
    left: -0.5rem;
  }
}

<div>
  <div
    class=""
  >
    <a
      aria-label="Open or close menu"
      class="c0"
      href="#"
      title="Open or close menu"
    >
      <svg
        aria-hidden="true"
        aria-label="Open menu"
        class="c1"
        fill="currentColor"
        focusable="false"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M0 0h24v24H0V0z"
          fill="none"
        />
        <path
          d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
        />
      </svg>
    </a>
    <div
      aria-hidden="true"
      class="c2"
    >
      <nav
        class="c3"
      >
        <div
          class="c4"
        >
          <h1
            class="c5 c6"
          >
            <a
              class="c7"
              href="/"
              target="_self"
            >
              <img
                alt="Otávio Miranda"
                src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614698965/Ubuntu_gel_by_Midge_Mantissa_Sinnaeve_7caea7ddb4.jpg"
              />
            </a>
          </h1>
        </div>
        <a
          class="c8"
          href="#target"
          target="_self"
        >
          O texto
        </a>
        <a
          class="c8"
          href="#link2"
          target="_blank"
        >
          o LINK 2
        </a>
      </nav>
    </div>
    <div
      class="c9"
    >
      <header
        class="c10"
      >
        <h1
          class="c5 c6"
        >
          <a
            class="c7"
            href="/"
            target="_self"
          >
            <img
              alt="Otávio Miranda - Uma descrição para o seu blog."
              src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614698965/Ubuntu_gel_by_Midge_Mantissa_Sinnaeve_7caea7ddb4.jpg"
            />
          </a>
        </h1>
        <div
          class="c11"
        >
          <h2
            class="c5 c12"
          >
            Otávio Miranda
          </h2>
          <p>
            Uma descrição para o seu blog.
          </p>
        </div>
      </header>
    </div>
    <div
      class="c13"
    >
      <article
        class=""
      >
        <div
          class="c14"
        >
          <header
            class="c15"
          >
            <h1
              class="c5 c16"
            >
              Vis nobis veritus maluisset id, nonumy adversarium
            </h1>
            <p
              class="c17"
            >
              Fugit aliquip erroribus eu eam, an mel vidit doctus. Vis nobis veritus maluisset id, nonumy adversarium neglegentur est ut. Tation persius pri eu, modo minim fastidii mei no. Ad tota pertinax definiebas eam.
            </p>
            <img
              alt="Vis nobis veritus maluisset id, nonumy adversarium"
              class="c18"
              src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614698965/Ubuntu_gel_by_Midge_Mantissa_Sinnaeve_7caea7ddb4.jpg"
            />
            <div
              class="c19"
            >
              <p>
                <span>
                  Por 
                </span>
                <a
                  href="/author/otavio-miranda"
                >
                  Otávio Miranda
                </a>
                <span
                  class="separator"
                >
                   | 
                </span>
                <time
                  datetime="2020-04-20"
                >
                  19 de abr. de 2020
                </time>
                <span
                  class="separator"
                >
                   | 
                </span>
                <span
                  class="categories"
                >
                  <span>
                    <a
                      href="/category/java-script"
                    >
                      JavaScript
                    </a>
                  </span>
                  <span>
                    <a
                      href="/category/react"
                    >
                      React
                    </a>
                  </span>
                </span>
              </p>
            </div>
          </header>
        </div>
        <div
          class="c20"
        >
          <div
            class="c21"
          >
            <p>
              Nostro 
              <strong>
                definitiones
              </strong>
               (
              <i>
                React
              </i>
              ) sed cu - 
              <u>
                Otávio Miranda
              </u>
              . Sea 
              <code>
                useEffect
              </code>
              , useState dicat fabulas 
              <a
                href="https://gist.github.com/luizomf/e7cf0c3d172801225cce694681ecb0ad"
              >
                consetetur
              </a>
               no, odio deseruisse assueverit mei eu, vis probo nonumes molestie id. Ipsum movet liberavisse an mea, diceret vituperata ne his, delenit apeirian quo ea. Imperdiet percipitur te eos. Per ludus aliquam cu. 
              <a
                href="https://gist.github.com/luizomf/e7cf0c3d172801225cce694681ecb0ad"
              >
                Eos ex tincidunt necessitatibus, timeam theophrastus mel cu
              </a>
              . La.
            </p>
            <pre>
              <code
                class="language-plaintext"
              >
                import { useEffect, useRef, useState } from 'react';

const useMyHook = (cb, delay = 1000) =&gt; {
  const savedCb = useRef();

  useEffect(() =&gt; {
    savedCb.current = cb;
  }, [cb]);

  useEffect(() =&gt; {
    const interval = setInterval(() =&gt; {
      savedCb.current();
    }, delay);

    return () =&gt; clearInterval(interval);
  }, [delay]);
};
              </code>
            </pre>
            <p>
              Qui eu 
              <i>
                mutat
              </i>
               exerci 
              <span
                style="background-color:hsl(60,75%,60%);"
              >
                dolores
              </span>
              , his no 
              <span
                style="color:hsl(270,75%,60%);"
              >
                repudiare
              </span>
               constituam 
              <s>
                accommodare
              </s>
              , vis ea sint 
              <u>
                tempor
              </u>
               eirmod. In mei eruditi moderatius, ex per iudico essent repudiandae. In vis vidisse menandri, an has repudiare disputationi delicatissimi. In has wisi justo postulant. Ei viderer ornatus mel, usu an partiendo tincidunt, ea eum brute ullamcorper.
            </p>
            <p>
              10
              <sup>
                2
              </sup>
              *5
              <sup>
                2
              </sup>
               - H
              <sub>
                2
              </sub>
              0
            </p>
            <p
              style="text-align:center;"
            >
              <strong>
                Center: 
              </strong>
              Abhorreant reformidans eum an, per equidem repudiandae no, ex cum elitr argumentum persequeris. In eam consul theophrastus delicatissimi, lobortis rationibus efficiendi mea ei, nam natum dolore scripserit ea. Sit partiendo eloquentiam ne. Sea no autem dolores, ridens gloriatur mel cu. Munere nullam epicuri his id, dicam definiebas deterruisset eu vim. Ea veritus deleniti usu, nam ei idque choro volumus, ex nisl aliquam bonorum eam.
            </p>
            <p>
              Fugit aliquip erroribus eu eam, an mel vidit doctus. Vis nobis veritus maluisset id, nonumy adversarium neglegentur est ut. Tation persius pri eu, modo minim fastidii mei no. Ad tota pertinax definiebas eam.
            </p>
            <p
              style="text-align:right;"
            >
              <strong>
                Right:
              </strong>
               Mel in lorem feugiat, tation dolore et eum. Id has utamur albucius molestiae. Dico justo maiestatis ei mei, labitur laboramus voluptatum nam ut. Saperet propriae nec an.
            </p>
            <p>
              Quod oblique mei ne, cetero delicatissimi sea cu, cu est harum dicit verterem. Tota forensibus ut usu, luptatum dignissim persecuti vel ex. Mea nostrud instructior cu, et referrentur theophrastus eum. Pro in iuvaret senserit. In persius explicari conceptam nec, in populo ponderum consequat vis. Gloriatur conclusionemque no qui, vitae nominati nec te, laoreet torquatos accommodare ex qui. Eu natum mediocritatem eos, cum ex alia oportere.
            </p>
            <p
              style="text-align:justify;"
            >
              <strong>
                Justify:
              </strong>
               Cu vis nonumes petentium disputationi, invidunt aliquando ex ius, ocurreret suscipiantur conclusionemque eu qui. Probo nemore ut est. Et eam dicta oratio eligendi. Quem dicta dolore vel et, at reque gloriatur definitionem sed.
            </p>
            <p>
              Nec simul ullamcorper at. Mea possit dolores persecuti te, ei quidam apeirian quo. Has quas corpora ne. Ius ad malis iusto delenit, sit ferri idque ex.
            </p>
            <blockquote>
              <p>
                Blockquote: Et sed legere rationibus, vitae evertitur ad eam. Eu amet vero officiis vis. Ad nullam pericula neglegentur sit, vix ei regione legendos argumentum. Mei an quas convenire, nobis dicunt mei ut. Pro no debet referrentur, ei est falli libris. Eu per summo salutandi hendrerit, salutatus democritum interesset nec ut. Wisi lobortis incorrupte ei mea, explicari interpretaris has et.
              </p>
            </blockquote>
            <p>
              A new line:
            </p>
            <hr />
            <p>
              Now keep going…
            </p>
            <figure
              class="image"
            >
              <a
                href="https://gist.github.com/luizomf/e7cf0c3d172801225cce694681ecb0ad"
              >
                <img
                  alt="O alt text da imagem"
                  src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614696630/Frozen_sunset_on_the_lake_by_Manuel_Arslanyan_9f9cd8ea10.jpg"
                />
              </a>
              <figcaption>
                Aqui está uma imagem
              </figcaption>
            </figure>
            <p>
              Munere euripidis an sed, usu ne consul soluta utamur. Bonorum equidem placerat eu cum, tantas quidam duo et. Pro soleat quaestio ne, ut nam wisi vocibus inciderint, ex his maiorum mnesarchum. Et est wisi primis, id vel odio alienum voluptatibus. Veri doctus offendit vim ea. Has nemore prompta inciderint ei, no sea expetendis dissentiet. Copiosae delicatissimi an sea, id cum agam clita nostro, tale tincidunt delicatissimi ei eos.
            </p>
            <figure
              class="image image-style-side"
            >
              <img
                alt="Uma imagem à direita"
                src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614696630/Frozen_sunset_on_the_lake_by_Manuel_Arslanyan_9f9cd8ea10.jpg"
              />
              <figcaption>
                Uma imagem à direita
              </figcaption>
            </figure>
            <p>
              Munere euripidis an sed, usu ne consul soluta utamur. Bonorum equidem placerat eu cum, tantas quidam duo et. Pro soleat quaestio ne, ut nam wisi vocibus inciderint, ex his maiorum mnesarchum. Et est wisi primis, id vel odio alienum voluptatibus. Veri doctus offendit vim ea. Has nemore prompta inciderint ei, no sea expetendis dissentiet. Copiosae delicatissimi an sea, id cum agam clita nostro, tale tincidunt delicatissimi ei eos.
            </p>
            <p>
              Munere euripidis an sed, usu ne consul soluta utamur. Bonorum equidem placerat eu cum, tantas quidam duo et. Pro soleat quaestio ne, ut nam wisi vocibus inciderint, ex his maiorum mnesarchum. Et est wisi primis, id vel odio alienum voluptatibus. Veri doctus offendit vim ea. Has nemore prompta inciderint ei, no sea expetendis dissentiet. Copiosae delicatissimi an sea, id cum agam clita nostro, tale tincidunt delicatissimi ei eos.
            </p>
            <p>
              Solum consul scripserit in duo, an sit postea denique percipit. Facer ignota consequat nec ei, idque etiam corrumpit sit ut. In mucius definitionem quo, vis an nisl diam appellantur, hendrerit comprehensam ea per. Vix saepe fierent ut.
            </p>
            <p>
              Cu mei summo copiosae. Et vim quaestio partiendo. Quot praesent imperdiet at duo. Sit probo causae ei. Ad ius nemore eruditi.
            </p>
            <p>
              Mei in qualisque disputando. Adipisci philosophia et sed, mel diceret patrioque sententiae ea, iriure posidonium consectetuer ad eam. Nobis reprimique duo in, eam corpora mandamus ex. In quod euismod eum, ne habemus cotidieque necessitatibus cum. Liber recteque duo no, ludus reprehendunt has an.
            </p>
            <p>
              Solum consul scripserit in duo, an sit postea denique percipit. Facer ignota consequat nec ei, idque etiam corrumpit sit ut. In mucius definitionem quo, vis an nisl diam appellantur, hendrerit comprehensam ea per. Vix saepe fierent ut.
            </p>
            <p>
              Cu mei summo copiosae. Et vim quaestio partiendo. Quot praesent imperdiet at duo. Sit probo causae ei. Ad ius nemore eruditi.
            </p>
            <p>
              Mei in qualisque disputando. Adipisci philosophia et sed, mel diceret patrioque sententiae ea, iriure posidonium consectetuer ad eam. Nobis reprimique duo in, eam corpora mandamus ex. In quod euismod eum, ne habemus cotidieque necessitatibus cum. Liber recteque duo no, ludus reprehendunt has an.
            </p>
            <p>
              Bullet:
            </p>
            <ul>
              <li>
                Mei in qualisque disputando. Adipisci philosophia et sed, mel diceret patrioque sententiae ea, iriure posidonium consectetuer ad eam. Nobis reprimique duo in, eam corpora mandamus ex. In quod euismod eum, ne habemus cotidieque necessitatibus cum. Liber recteque duo no, ludus reprehendunt has an.
              </li>
              <li>
                iriure posidonium consectetuer ad eam.
              </li>
              <li>
                Nobis reprimique duo in, eam corpora mandamus
              </li>
              <li>
                ex. In
              </li>
              <li>
                quod euismod
              </li>
              <li>
                eum, ne habemus cotidieque necessitatibus cum.
              </li>
              <li>
                Liber recteque duo no, ludus reprehendunt has an.
              </li>
            </ul>
            <p>
              Now ordered:
            </p>
            <ol>
              <li>
                Item 1
              </li>
              <li>
                Item
              </li>
              <li>
                Item
              </li>
              <li>
                Item
              </li>
            </ol>
            <p>
              A think that's it…
            </p>
            <figure
              class="table"
            >
              <table>
                <thead>
                  <tr>
                    <th>
                      Title 1
                    </th>
                    <th>
                      Title 1
                    </th>
                    <th>
                      Title 1
                    </th>
                    <th>
                      Title 1
                    </th>
                    <th>
                      Title 1
                    </th>
                    <th>
                      Title 1
                    </th>
                    <th>
                      Title 1
                    </th>
                    <th>
                      Title 1
                    </th>
                    <th>
                      Title 1
                    </th>
                    <th>
                      Title 1
                    </th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                  </tr>
                  <tr>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                  </tr>
                  <tr>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                  </tr>
                </tbody>
              </table>
            </figure>
            <p>
              A video:
            </p>
            <figure
              class="media"
            >
              <oembed
                url="https://www.youtube.com/watch?v=oLmRV25eQvo"
              />
            </figure>
            <p>
              The end.
            </p>
          </div>
        </div>
      </article>
      <div
        class="c22"
      >
        <p
          class="c23"
        >
          tags:
          <span>
            <a
              href="/tag/use-effect"
            >
              useEffect
            </a>
          </span>
          <span>
            <a
              href="/tag/use-state"
            >
              useState
            </a>
          </span>
        </p>
      </div>
    </div>
    <div
      class="c24"
    >
      <footer
        class="c25"
      >
        <div
          class="c21"
        >
          <p>
            O texto da footer
          </p>
        </div>
      </footer>
    </div>
    <a
      aria-label="Go to top"
      class="c26"
      href="#"
      title="Go to top"
    >
      <svg
        aria-hidden="true"
        class="c1"
        fill="currentColor"
        focusable="false"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M0 0h24v24H0V0z"
          fill="none"
        />
        <path
          d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6 1.41 1.41z"
        />
      </svg>
    </a>
  </div>
</div>
`;
